### PR TITLE
Package ipyurl

### PR DESCRIPTION
**Pull request by davidbrochart**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by davidbrochart** [comment]
Trigger CI

File: recipes/ipyurl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{% set version = "0.1.2" %}
+{% set sha256 = "9831f53ed6c564b59d72a047e627785139069e4bc0311d3393fd9fc6f52389bb" %}
+
+package:
+  name: ipyurl
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/i/ipyurl/ipyurl-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```
Comment (xhochy) [inline recipes/ipyurl/meta.yaml:10]: Please omit the Jinja here.
```suggestion
  sha256: 9831f53ed6c564b59d72a047e627785139069e4bc0311d3393fd9fc6f52389bb
```

File: recipes/ipyurl/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - sysroot_linux-64 2.17  # [linux64]
```
Comment (xhochy) [inline recipes/ipyurl/meta.yaml:20]: Please remove that in the feedstock again and do the mentioned changes to `conda-forge.yaml`.

File: recipes/ipyurl/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```
Comment (xhochy) [inline recipes/ipyurl/meta.yaml:19]: sysroot should be enough
```suggestion
```

**Comment by davidbrochart** [comment]
OK to merge now @xhochy ?

File: recipes/ipyurl/meta.yaml (hunk 2 above)
Comment (davidbrochart) [reply recipes/ipyurl/meta.yaml]: You mean, not here in the recipe but when the feedstock is published, right?

**Comment by davidbrochart** [comment]
@conda-forge/help-python, it's ready to be merged.

**Pull request merged**